apps/examples/testcase/le_tc/kernel/tc_sched.c

```diff
@@ -387,14 +387,16 @@ static void tc_sched_waitpid(void)
 */
 static void tc_sched_sched_gettcb(void)
 {
-	struct tcb_s *tcb;
+	int fd;
+	int pid;
+	fd = tc_get_drvfd();
 
-	tcb = sched_gettcb(INVALID_PID);
-	TC_ASSERT_EQ("sched_gettcb", tcb, NULL);
+	pid = ioctl(fd, TESTIOC_GET_TCB_PID, (unsigned long)INVALID_PID);
```

Comment:
TESTIOC_GET_TCB_PID -> TESTIOC_GET_PID

Reply:
okay. thanks

Reply:
1. Original code is `sched_gettcb`, not `getpid`. Why does it change to get pid?
2. Command is GET_PID, but you give INVALID_PID as 3rd argument. What is it???